README.md

```diff
@@ -41,12 +41,12 @@ data = np.array(
     ],
 )
 
-event_timestamps = np.array([0.25, 5.0, 12.25])  # The timestamps to which we align the counts
+timestamps = np.array([0.25, 5.0, 12.25])  # The timestamps to which we align the counts
 milliseconds_from_event_to_first_bin = -50.0  # The first bin is 50 ms before the event
 bin_width_in_milliseconds = 100.0  # Each bin is 100 ms wide
 binned_aligned_spikes = BinnedAlignedSpikes(
     data=data,
-    event_timestamps=event_timestamps,
+    timestamps=timestamps,
```

Comment:
why did you change this? I liked it before. It disambiguates this from the timestamps of the bins

Reply:
Interesting point, I missed that it was useful to disambiguate that way. The reason I changed this was to make it consistent with most pynwb objects that have a timestamp attribute but I can revert.

Reply:
I do think it would make sense to call this variable `timestamps` if the object inherits from `TimeSeries`. I noticed you decided not to do that here. Did you consider inheriting from `TimeSeries`? Why did you make the decision not to. I'm on the fence about it myself so I'm curious what you think.

Reply:
I thought about it but it has some attributes that I don't think make a lot of sense here. continuity, conversion, offset, etc. I think in some cases it might make sense to have a rate instead of timestamps but that also seemed like a complication.

Reply:
yeah I would agree, let's not use `TimeSeries`. I think that is a good reason to use `event_timestamps`, so this isn't mistaken for a `TimeSeries`

Reply:
OK, let me go back to this.

Reply:
Changed.